save-cloud-common/src/jvmMain/kotlin/com/saveourtool/save/utils/IdentitySourceAwareUserDetails.kt

```diff
@@ -1,7 +1,13 @@
 package com.saveourtool.save.utils
 
+import org.slf4j.LoggerFactory
 import org.springframework.security.core.authority.SimpleGrantedAuthority
-import org.springframework.security.core.userdetails.User
+import org.springframework.security.core.userdetails.UserDetails
+import org.springframework.security.core.userdetails.UsernameNotFoundException
+import reactor.core.publisher.Mono
+import reactor.kotlin.core.publisher.switchIfEmpty
+
+private val logger = LoggerFactory.getLogger(object {}.javaClass.enclosingClass::class.java)
```

Comment:
Can be simplified as either

```kotlin
private val logger = LoggerFactory.getLogger({}.javaClass)
```

or, better

```kotlin
private val logger = getLogger<IdentitySourceAwareUserDetails>()
```

Reply:
Thanks, fixed